api/routers/legacy.py

```diff
@@ -124,7 +124,8 @@ async def sql_insert_player(player_name):
 
     await execute_sql(sql_insert, param=param)
     player = await sql_get_player(player_name)
-    return player
+
+    return player[0] if player is not None else None
```

Comment:
i think this is more readable?
`return None if player is None else player[0]`
or 
`return player[0] if not player is None else None`